### ISSUE TITLE
Feature Request: Use Scene View as Default tab, not Drafts View

### ISSUE THREAD

**Issue by chrisgrieser** [kevboh/longform#141]
**Describe the solution you'd like**
More a quality-of-life feature, but I find it quite annoying that every time I open up the Longform Sidebar, it opens the Drafts Tab. Considering that since Longform 2.0 Drafts are a less important feature, and you can easily make due without using Drafts, it's a bit annoying the plugin still constantly opens the Drafts Tab by default.

**Describe alternatives you've considered**
I cannot think of one.

**Comment by kevboh** [comment]
By Drafts do you mean Project? Makes sense for scene projects, yes. TBH I thought this was how it already behaved—defaults to Scenes for scene-based projects, but evidently not!

**Comment by chrisgrieser** [comment]
ah yes, I meant the projects tab!

**Comment by chrisgrieser** [comment]
Considering I use the plugin a lot (and also realized that I created more than two dozen issues at this repo 🙈), I might as well make some PRs, if you want?

I was thinking to start small, and this one might do. Could you point me to the relevant sections of the code for me to get started here?

**Comment by b-camphart** [comment]
@chrisgrieser possibly [here](https://github.com/kevboh/longform/blob/main/src/view/explorer/ExplorerView.svelte)?  That's where the initial selected tab is determined.

**Comment by chrisgrieser** [comment]
@b-camphart Nope, tried everything in that file. Though, looking through that file gave me the idea to grep for `selectedTab` through the repo, which after some trials and error led me to this line: https://github.com/kevboh/longform/blob/8fe5f6d5e83f0bb2f6cdd9d48d42087bfe44c8b4/src/view/stores.ts#L23

and in fact, it looks like it that line that controls which tab is opened by default. 😊
